k8s/README.md

```diff
@@ -0,0 +1,313 @@
+# Running RaftKeeper, A Distributed System Coordinator
+
+## Prerequisite
+
+You must have a cluster with at least four nodes, and each node requires at least 2 CPUs and 4 GiB of memory. 
```

Comment:
1. four nodes -> three nodes
2. must -> recommend

Reply:
Done in most recent commit.